api/api/settings.py

```diff
@@ -167,14 +169,37 @@
 CATALOG_ZONE = 'catalog.internal'
 
 # Celery
+# TODO upgrade settings https://docs.celeryproject.org/en/stable/userguide/configuration.html#new-lowercase-settings
```

Comment:
But the page behind the link says that they won't accept uppercase settings anymore. Our's are all uppercase?

Reply:
That's if you configure Celery directly. If you configure it through Django, [things remain uppercase](https://docs.celeryproject.org/en/stable/history/whatsnew-4.0.html#latentcall-django-admonition):

> If you’re loading Celery configuration from the Django settings module then you’ll want to keep using the uppercase names.
> 
> You also want to use a `CELERY_` prefix so that no Celery settings collide with Django settings used by other apps.

Although it's a bit hard to find in the docs, it's pretty clear. Maybe we should replace the TODO comment with a comment linking to that section specifically.

Reply:
done